### ISSUE TITLE
Create common labels for priority bugs across repositories

### ISSUE THREAD

**Issue by SantoshNC68** [eiffel-community/community#36]
### Description
We need to have different label options to say the priority of the bug, so that when a user creates a bug can place these labels with options ranging from low to blocker (something similar to how JIRA provides)

### Motivation
We have lots of issues on Github, but we never know what are the priority ones that needs to be addressed. By adding the priority labels it will be easier to identify what needs to be taken care of first.

### Benefits
Easier planning of what to prioritize.

**Comment by e-pettersson-ericsson** [comment]
I’ve tried searching how other projects handle this on GitHub and I found this article: https://medium.com/@dave_lunny/sane-github-labels-c5d2e6004b63

The labels we currently have only state what type of issue, not priority. If we want to visualize priority on our issues this could be the way to go:

* Priority: Low
* Priority: Medium
* Priority: High

I don’t know how many we want to have but I hope you get the idea with this. Then it becomes easy to tell at first glance what the label means. However, I think it’s important to remember that these priority labels are only within the context of that repository (not in comparison to work done in other repositories). 

Should we also align these existing labels with the same naming conventions?

* Type: bug
* Type: enhancement
* Type: documentation

The rest of the existing labels (mostly default ones by GitHub) could be used together 
with the others, such as “help wanted” or “question” etc…

**Comment by SantoshNC68** [comment]
Yes, we only have some labels but the labels can be descriptive like the way you say, priority:low, high and medium are good, and the other labels can have differentiation, either type or by their name itself.
